protobuf/flow/executiondata/executiondata.proto

```diff
@@ -221,7 +222,10 @@ message GetRegisterValuesRequest {
   uint64 block_height = 1;
 
   // Register IDs of the Ledger.RegisterID format with an owner and key.
-  repeated bytes register_ids = 2;
+  repeated bytes register_ids = 2; // deprecated!
+
+  // Register IDs of the Ledger.RegisterID format with an owner and key.
+  repeated  entities.RegisterID register_ids_v2 = 3;
```

Comment:
since this proto was never implemented, I think it's fine to just update the original
```suggestion
  // Register IDs of the Ledger.RegisterID format with an owner and key.
  repeated  entities.RegisterID register_ids = 2;
```